test/tall/regression/0700/0707.stmt

```diff
@@ -1,4 +1,4 @@
 >>> (indent 4) (fix optional-new)
     sink.write('FILE ACCESSED ${new DateTime.now()}\n');
 <<<
-    sink.write('FILE ACCESSED ${DateTime.now()}\n');
\ No newline at end of file
+    sink.write('FILE ACCESSED ${new DateTime.now()}\n');
```

Comment:
Did this test run with different arguments before?
Or have we not implemented fixes in the tall style?

Reply:
Oh, good catch! I didn't notice this when I was looking at the diff.

The new style doesn't implement fixes and won't implement them. We'll be removing `dart format --fix` when the new style ships (since `dart fix` can now do all the same fixes and plenty more). It looks like there are only two regression tests (both coincidentally in the 0700s) that use fixes, so I'll just remove them from the tall tests since they aren't relevant.